pyproject.toml

```diff
@@ -11,7 +11,7 @@ description-file = "README.rst"
 keywords = "htcondor classad python"
 classifiers = ["License :: OSI Approved :: MIT License"]
 requires = [
-    "pyparsing"
+    "pyparsing >= 2.4.1"
```

Comment:
I would feel more confident limiting upper version as well  by using ``pyparsing ~= 2.4.1`` or ``pyparsing >= 2.4.1 <4.0.0``.

@eileen-kuehn might want to comment on this too.